README.md

```diff
@@ -1,90 +1,39 @@
-# Serverless SaaS - Reference Solution
+# aws-saas-factory-ref-solution-serverless-saas
 
-**[Feedback & Feature request](https://www.pulse.aws/survey/EHE3TICQ)** | **[Documentation](DOCUMENTATION.md)**
-
-We have also created a workshop that you can use as a reference to understand this reference solution in a step-by-step fashion. Workshop is available [here](https://github.com/aws-samples/aws-serverless-saas-workshop).
```

Comment:
any reason removing the workshop reference?

Reply:
Placed workshop reference back in again.